cmd/scollector/collectors/dotnet_windows.go

```diff
@@ -245,3 +259,163 @@ type Win32_PerfRawData_NETFramework_NETCLRMemory struct {
 	ProcessID                          uint32
 	PromotedFinalizationMemoryfromGen0 uint32
 }
+
+// The PID must be extracted from the Name field, and we get the name from the Pid
+var dotNetSQLPIDRegex = regexp.MustCompile(`.*\[(\d+)\]$`)
+
+func c_dotnet_sql() (opentsdb.MultiDataPoint, error) {
+	var dst []Win32_PerfRawData_NETDataProviderforSqlServer_NETDataProviderforSqlServer
+	err := queryWmi(dotnetSQLQuery, &dst)
+	if err != nil {
+		return nil, err
+	}
+	var svc_dst []Win32_Service
+	var svc_q = wmi.CreateQuery(&svc_dst, `WHERE Started=true`)
+	err = queryWmi(svc_q, &svc_dst)
+	if err != nil {
+		return nil, err
+	}
+	var iis_dst []WorkerProcess
+	iis_q := wmi.CreateQuery(&iis_dst, "")
+	err = queryWmiNamespace(iis_q, &iis_dst, "root\\WebAdministration")
+	if err != nil {
+		iis_dst = nil
+	}
+	var md opentsdb.MultiDataPoint
+	// We add the values of multiple pools that share a PID, this could cause some counter edge cases
+	pidToRows := make(map[string][]int)
```

Comment:
Please explain what the counters represent in the value arrays. Are they fixed length or variable?